schema/me.js

```diff
@@ -0,0 +1,32 @@
+import date from './fields/date';
+import gravity from '../lib/loaders/gravity';
+import Profile from './profile';
+import {
+  GraphQLString,
+  GraphQLObjectType,
+} from 'graphql';
+
+const Me = new GraphQLObjectType({
+  name: 'Me',
+  fields: {
+    id: {
+      type: GraphQLString,
+    },
+    created_at: date,
+    email: {
+      type: GraphQLString,
+    },
+    profile: {
+      type: Profile.type,
+      resolve: ({ default_profile_id }) =>
+        gravity(`profile/${default_profile_id}`),
+    },
+  },
+});
+
+export default {
+  type: Me,
+  resolve: (root, options, { rootValue: { accessToken } }) => {
```

Comment:
So when composing a GraphQL you pass the `accessToken` in a `rootValue` param? Or is it passed through the headers? Trying to wrap my head around what this looks like from the client-perpective.